templates/vhost.j2

```diff
@@ -2,6 +2,7 @@
 {% if item.server_name_redirect is defined %}
 server {
     listen       {{ item.listen | default('80') }};
+    listen [::]:{{item.listen | default('80') }};
```

Comment:
Should be spaced as     `listen       [::]:{{item.listen | default('80') }};` to follow the indent in that block

Reply:
Addressed this.